charts/velero/templates/servicemonitor.yaml

```diff
@@ -12,11 +15,15 @@ metadata:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  namespaceSelector:
+    matchNames:
+      - {{ .Release.Namespace }}
```

Comment:
I'm not sure should it be limited to _only_ monitor the velero release namespace or not.

Reply:
What other namespace are you thinking about?